frontend/http/frontend.go

```diff
@@ -39,7 +39,11 @@ var promResponseDurationMilliseconds = prometheus.NewHistogramVec(
 func recordResponseDuration(action string, af *bittorrent.AddressFamily, err error, duration time.Duration) {
 	var errString string
```

Comment:
Do you think it'd be slower to change this to `errString := "internal error"` and drop the `else` case?